### ISSUE TITLE
Is `rs` supposed to work after the app crashed?

### ISSUE THREAD

**Issue by i5o** [remy/nodemon#1490]
- `nodemon -v`: 1.18.7
- `node -v`: v11.3.0
- Operating system/terminal environment: MacOS Mojave 10.14.2 / iterm2
- Command you ran: nodemon index.js

## Question
Hello. Is nodemon supposed to restart if I write `rs` after a script fails?
If that is the case, this isn't working on my system.
If that isn't, that would be a cool feature request.
![image](https://user-images.githubusercontent.com/2650479/49936349-78da3400-feb2-11e8-89cc-b2e2d5085cb9.png)

**Comment by remy** [comment]
Yes it should.

Please can you provide full details on how to replicate (with a fully parred down example case).

**Comment by i5o** [comment]
@remy 	thanks for replying.
The code I had is basically a express server, and that's it.
When it fails "rs" doesn't get detected.

This is the example code that will just reproduce the error:
```
const express = require('express');
var app = express();
app.listen('7003', '0.0.0.0', function() {
  throw 'hey'
});
```

Save it to index.js and then `nodemon index.js`
![image](https://user-images.githubusercontent.com/2650479/49950027-c3b97300-fed5-11e8-9c79-be498030c84c.png)

**Comment by remy** [comment]
Can you give it a try with the latest nodemon? 1.18.8

**Comment by i5o** [comment]
Not working

![image](https://user-images.githubusercontent.com/2650479/49962866-31c16280-fef5-11e8-9152-2860141f19e9.png)

**Comment by remy** [comment]
Fixed in nodemon@1.18.9

**Comment by i5o** [comment]
Can confirm it's fixed on 1.18.9 :)

![image](https://user-images.githubusercontent.com/2650479/50018897-8de8bd00-ffaf-11e8-9d21-836d5ef45732.png)


Thanks. 😸
